### ISSUE TITLE
check_links fails on testbuchen.de

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#2132]
## Where to find the issue

[check_links.js](https://github.com/corona-warn-app/cwa-website/blob/master/cypress/integration/check_links.js)

## Describe the issue

[check_links.js](https://github.com/corona-warn-app/cwa-website/blob/master/cypress/integration/check_links.js) produces a 403 (http forbidden) error when checking the link 

https://testbuchen.de/#/?zoom=0&lat=47.71401323721353&lng=8.66960999999999

When opened manually, this link shows no error.

## Steps to reproduce

`npm run test:open`
select `check_links.js`

When the test fails, then view the file `cypress/logs/broken_links_result.txt` which contains:

```
==================== Broken links ====================

[RESPONSE 403] https://testbuchen.de/#/?zoom=0&lat=47.71401323721353&lng=8.66960999999999 on 'http://localhost:8000/de/blog/2021-05-11-how-to-rapid-test-integration' 
[RESPONSE 403] https://testbuchen.de/#/?zoom=0&lat=47.71401323721353&lng=8.66960999999999 on 'http://localhost:8000/de/blog/2021-05-02-corona-warn-app-version-2-1' 
[RESPONSE 403] https://testbuchen.de/#/?zoom=0&lat=47.71401323721353&lng=8.66960999999999 on 'http://localhost:8000/en/blog/2021-05-11-how-to-rapid-test-integration' 
[RESPONSE 403] https://testbuchen.de/#/?zoom=0&lat=47.71401323721353&lng=8.66960999999999 on 'http://localhost:8000/en/blog/2021-05-02-corona-warn-app-version-2-1' 
```

I don't know what is causing this error and this is a new problem.

**Comment by MikeMcC399** [comment]
It seems that the site testbuchen.de has added network security to stop it being automatically accessed.

![Error 1020](https://user-images.githubusercontent.com/66998419/143623929-b05ac688-a0a5-4238-b349-214cb94a0415.png)

So we need some way in check_links.js to make an exception for websites like testbuchen.de which don't allow link checking automatically.

This is similar to a problem we had with markdown-check-links on the cwa-documentation site (see https://github.com/corona-warn-app/cwa-documentation/issues/542#issuecomment-795571682).

I solved that by adding:
```
<!-- markdown-link-check-disable --> disables markdown link check. 
<!-- markdown-link-check-enable --> reenables markdown link check.
```
see PR https://github.com/corona-warn-app/cwa-documentation/pull/545.

**Comment by MikeMcC399** [comment]
Also the CWA Open Source Team might like to contact the support team of https://testbuchen.de to check if they could remove this link test restriction.

The Impressum of the above site https://www.no-q.info/impressum/ includes e-mail address support@vertical-life.info.

**Comment by MikeMcC399** [comment]
This issue has been fixed by PR #2136.
Thanks to @brifemu!
